scalding-date/src/test/scala/com/twitter/scalding/DateTest.scala

```diff
@@ -97,6 +98,12 @@ class DateTest extends WordSpec {
       assert(rd1 >= rd1)
       assert(rd2 >= rd2)
     }
+    "be able to compare with before() and after() with TimeZone in context" in {
+      //implicit val tz: TimeZone = TimeZone.getDefault
```

Comment:
Uncommented. It would be failing even if I would remove the comment, but then the validness of the test would depend on that implicit tz on the top level (which someone someday might decide to remove, rendering my test trivial).